spec/support/initializers/capybara.rb

```diff
@@ -21,7 +21,16 @@
   end
 
   config.before(:each, type: :system, js: true) do
-    driven_by :headless_chrome
+    if ENV["SELENIUM_URL"].present?
+      # Support use of selenium-docker
+      Capybara.server_host = "0.0.0.0"
+      ip = Socket.ip_address_list.detect(&:ipv4_private?).ip_address
+      host! "http://#{ip}:#{Capybara.server_port}"
```

Comment:
I generally prefer building URI's with appropriate tools, but don't let that stop you form merging:

```suggestion
      URI::HTTP.build(host: ip, port: Capybara.server_port).to_s
```

Reply:
that looks much neater than piecing together strings.